lib/migration.pm

```diff
@@ -207,4 +209,19 @@ sub set_scc_proxy_url {
     save_screenshot;
 }
 
+# Enable ZYPP_SINGLE_RPMTRANS=1 if the OS have
```

Comment:
It is better add Jira feature ID and title in the description of the function for other to refer to.

Reply:
done